### ISSUE TITLE
Error when declaring supports: AttributeError: module 'collections' has no attribute 'Iterable'

### ISSUE THREAD

**Issue by ajossorioarana** [ritchie46/anaStruct#90]
Whenever I declare supports, for example with the following code,
```python
ss.add_support_hinged(node_id=1)
```

The following error is raised: **_"AttributeError: module 'collections' has no attribute 'Iterable'"_**

As stated in [this StackOverflow post](https://stackoverflow.com/questions/59809785/i-get-a-attributeerror-module-collections-has-no-attribute-iterable-when-i): 

> "collections.Iterable is deprecated. Replace it with collections.abc.Iterable."

So I did that in the files that appeared on the log. Now I can't remember what files appeared, but if anyone stumbles upon this error, this should do the trick.

**Comment by paters06** [comment]
Hi. I have found the same bug and follows your suggestion. The files where I made the modification had these relative paths:

(I'm working on Fedora 35, in the case this is necessary. These directories are located in the home folder)

`/.local/lib/python3.10/site-packages/anastruct/fem/system.py`
`/.local/lib/python3.10/site-packages/anastruct/basic.py`

Sorry if my english is not very well understood. Have a nice day
